lib/tasks/build.js

```diff
@@ -61,5 +62,9 @@ module.exports = Task.extend({
 
         throw err;
       });
+  },
+
+  onInterrupt: function() {
+    return RSVP.Promise.reject(1);
```

Comment:
Does `onInterrupt` actually have any ability to control exit code?  I think any handler that rejects exist with code 1: https://github.com/ember-cli/capture-exit/blob/2a2ba681721f65f7213dfc67e621aaa922eb8320/index.js#L97

That said it does seem reasonable to allow handlers to reject with an exit code, although it raises the issue of what to do when you have `process.exit(a)` and a handler that rejects `b`.  Likely the right code in this case is `a`.

Anyway I bring it up here but the right place for hashing it out will be on https://github.com/ember-cli/capture-exit/pull/18